backend/java/src/main/java/org/unipept/storage/PeptideLoaderData.java

```diff
@@ -20,84 +23,109 @@
+                    + " Error creating sequenceIds database.");
             e.printStackTrace();
             System.exit(1);
         }
-    }
 
-    /**
-     * creates all prepared statements used in this class.
-     */
-    private void prepareStatements() {
+        /* Opening CSV files for writing. */
         try {
-            containsSequence = connection
-                    .prepareStatement("SELECT id FROM sequences WHERE `sequence` = ?");
-            addSequence = connection.prepareStatement(
-                    "INSERT INTO sequences (`sequence`) VALUES (?)",
-                    Statement.RETURN_GENERATED_KEYS);
-            addUniprotEntry = connection
-                    .prepareStatement(
-                            "INSERT INTO uniprot_entries (`uniprot_accession_number`, `version`, `taxon_id`, `type`, `protein`) VALUES (?,?,?,?,?)",
-                            Statement.RETURN_GENERATED_KEYS);
-            addPeptide = connection
-                    .prepareStatement("INSERT INTO peptides (`sequence_id`, `uniprot_entry_id`, `original_sequence_id`, `position`) VALUES (?,?,?,?)");
-            addLineage = connection
-                    .prepareStatement("INSERT INTO lineages (`taxon_id`) VALUES (?)");
-            addInvalidLineage = connection
-                    .prepareStatement("INSERT INTO lineages (`taxon_id`, `superkingdom`, `kingdom`, `subkingdom`, `superphylum`, `phylum`, `subphylum`,`superclass`, `class`, `subclass`, `infraclass`, `superorder`, `order`, `suborder`, `infraorder`, `parvorder`, `superfamily`, `family`, `subfamily`, `tribe`, `subtribe`, `genus`, `subgenus`, `species_group`, `species_subgroup`, `species`, `subspecies`, `varietas`, `forma`) VALUES (?, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1)");
-            addRefseqRef = connection
-                    .prepareStatement("INSERT INTO refseq_cross_references (`uniprot_entry_id`, `protein_id`, `sequence_id`) VALUES (?,?,?)");
-            addEMBLRef = connection
-                    .prepareStatement("INSERT INTO embl_cross_references (`uniprot_entry_id`, `protein_id`, `sequence_id`) VALUES (?,?,?)");
-            addGORef = connection
-                    .prepareStatement("INSERT INTO go_cross_references (`uniprot_entry_id`, `go_id`) VALUES (?,?)");
-            addECRef = connection
-                    .prepareStatement("INSERT INTO ec_cross_references (`uniprot_entry_id`, `ec_id`) VALUES (?,?)");
-            lineageExists = connection
-                    .prepareStatement("SELECT COUNT(*) AS aantal FROM lineages WHERE `taxon_id` = ?");
-            getTaxon = connection
-                    .prepareStatement("SELECT rank, parent_id, valid_taxon FROM taxons WHERE id = ?");
-        } catch (SQLException e) {
+            uniprotEntries = new CSVWriter("uniprotEntries.tsv", "uniprot_accession_number", "version", "taxon_id", "type", "protein");
```

Comment:
this file should be named `uniprot_entries.tsv`


_[Original comment](https://github.ugent.be/unipept/unipept/pull/424#discussion_r2074) by @bmesuere on Fri Jan 16 2015 at 14:34._